src/helpers/AuthenticateToken.js

```diff
@@ -16,7 +18,31 @@ export default class AuthenticateToken {
     */
   static signToken(data) {
     const { lastName, firstName, ...newData } = data;
-    const token = jwt.sign(newData, process.env.JWT_KEY);
+    const token = jwt.sign(newData, process.env.JWT_KEY, { expiresIn: 60 * 60 * 24 * 7 });
```

Comment:
It is good to generate a token that expires after a short period, for instance `1W, 1D, 1M`